### DISCUSSION TITLE
Breaklines in abstract or description ?

### DISCUSSION THREAD

**Discussion by kikislater** [eblondel/geoflow#277] [Q&A] [answered]
Hello,

Breaklines are removed when upload from metadata to GN, is there a way to upload some breaklines ? Or do I have to look in Geonetwork or Geoflow side ? 
it seems to be on GN side but confirmation will help please : https://github.com/geonetwork/core-geonetwork/blob/main/services/src/main/java/org/fao/geonet/api/mapservers/MapServersApi.java#L519=

Sylvain

**Comment by eblondel** [comment]
Did you check the ISO 19139 metadata that is produced and stored locally in your workflow execution directory? Is it well kept in your ISO 19139 (normally yes, this was solved long time ago)? If yes, then we may need to look into GN side.

**Comment by kikislater** [comment]
Ok good point, thank you. 
If you put a comma or a colon then breakline it's removed by geoflow ... But then removing comma and/or colon produce correct xml but uploaded to GeoNetwork remove all breaklines

**Reply by eblondel** [reply]
You can double quote the abstract text to avoid cutting text, did you try it?

**Reply by kikislater** [reply]
yes it works, I was trying but without checking in iso 19139, So we can close here

**Reply by eblondel** [reply]
But do you still have issues within GN? Do they remove breaklines from texts?

**Reply by kikislater** [reply]
Yes but it's not on GeoFlow side now : 
Temporary links, it will be remove later ... 
https://docs.google.com/spreadsheets/d/1CPOWTycrA0i76rTk-2TMxTyNhim7sxdJb3DuH41WG08/edit?usp=sharing
https://gn.marbec-tools.ird.fr/geonetwork/srv/fre/catalog.search#/metadata/tiles_S2CHAIN-Sylvainv2

**Reply by kikislater** [reply]
and finally, importing xml generated by GeoFlow without using GN API works : https://gn.marbec-tools.ird.fr/geonetwork/srv/fre/catalog.search#/metadata/ebc359d1-3fe9-49bf-be3b-741a4a8486b8

**Comment by kikislater** [comment, marked as answer]
To resume, answer is here : https://github.com/eblondel/geoflow/discussions/277#discussioncomment-3220666

- Use double quote to avoid comma and colon + brealine to be removed by GeoFlow
- Check xml generated by Geoflow

**Reply by kikislater** [reply]
@eblondel Could you suggest or create a block in documentation to describe `entities` and add this kind of remarks to avoid such behaviors ? 
I'll contribute to fill documentation in that way
